weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/datetime.datetime/DateTimeView.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import {LoadingDots} from '../../../../../LoadingDots';
+import {Timestamp} from '../../../../../Timestamp';
+import {NotApplicable} from '../../NotApplicable';
+import {useWFHooks} from '../../pages/wfReactInterface/context';
+import {CustomWeaveTypePayload} from '../customWeaveType.types';
+
+type DateTimeTypePayload = CustomWeaveTypePayload<
+  'datetime.datetime',
+  {'obj.json': string}
+>;
+
+type DateTimeViewProps = {
+  entity?: string;
+  project?: string;
+  data: DateTimeTypePayload | string | null | undefined;
+};
+
+export const DateTimeView = ({entity, project, data}: DateTimeViewProps) => {
+  const {useFileContent} = useWFHooks();
+  const datetimeKey = 'obj.json';
+  const isCustomType = data != null && typeof data === 'object';
+  const fileDigest =
+    isCustomType && datetimeKey in data.files ? data.files[datetimeKey] : '';
+  const datetimeBinary = useFileContent(
+    entity || '',
+    project || '',
+    fileDigest,
+    {skip: !isCustomType || !(datetimeKey in data.files)}
+  );
+
+  // Handle null/undefined cases early
+  if (data == null) {
+    return <NotApplicable />;
+  }
+
+  // Handle direct ISO string input
+  if (typeof data === 'string') {
+    const stringTimestamp = new Date(data).getTime() / 1000;
+    return <Timestamp value={stringTimestamp} format="relative" />;
```

Comment:
I think we shouldn't use relative time for this by default - can be disconcerting for user to see their call output appear to change its value.

A few ideas for follow up:
* We could make the component a little fancier like we do in `CompareGridCellValueTimestampProps` - user could click on the value and have it cycle through the absolute string representation, relative time, unix epoch int etc.
* We should probably do something to make it more visually obvious that what was logged was a datetime object and not a string representation - maybe a little icon prefix like this:
<img width="59" alt="Screenshot 2025-02-20 at 1 27 50 PM" src="https://github.com/user-attachments/assets/66b27118-d507-4852-aa17-1908ebbf359e" />